src/PerformanceTests/Utils/Statistics.cs

```diff
@@ -76,12 +69,6 @@ public static IDisposable Initialize(Permutation permutation)
     {
         this.permutation = permutation;
         process = Process.GetCurrentProcess();
-        privateBytesCounter = new PerformanceCounter("Process", "Private Bytes", process.ProcessName);
-        perfCountersTimer = new Timer(state =>
```

Comment:
Why removed the timer? Would that be replaced with a call to `process.PrivateMemorySize64`?

Reply:
not sure I understand the comment. Do you suggest to keep the timer to calculate an average of `process.PrivateMemorySize64`?

Reply:
That was the existing behavior. I don't know of how much use it is. @dvdstelt What do you think? Shall we just drop this? The peak memory usage is probably already a good indicator as most tests would just run 1 minute.